tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
What @kidder said is all I'm arguing for. For abstract base classes you can define _an_ equivalence operator if the derived classes are copyable, but that does not mean it is the right equivalence operator for the base class.